DiceKit/FrequencyDistribution.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  FrequencyDistribution.swift
+//  DiceKit
+//
+//  Created by Brentley Jones on 7/24/15.
+//  Copyright © 2015 Brentley Jones. All rights reserved.
+//
+
+import Foundation
+
+public struct FrequencyDistribution: Equatable {
+    
+    public typealias Value = Int
+    public typealias Outcome = Double
+    public typealias OutcomesPerValue = [Value: Outcome]
```

Comment:
Yeah, it was only after fully understanding `FrequencyDistribution` I was able to think of it as a Frequency Distribution, but in a stats course they would be named just like how you said. The value of a die is kind of like an outcome of an experiment (roll the die, record the result) that has a certain frequency. At least, that's how it would be described. So if we are willing to go back and change it so that stats people can understand it, I am pretty sure we would rename `Outcome` as `Frequency`, and we could also rename `Value` as `Outcome` (or something along these lines). I think having our terms match the terms in a highly referenced source, such as say the [Wikipedia](https://en.wikipedia.org/wiki/Frequency_distribution) article ("**frequency distribution** is a table that displays the **frequency** of various **outcomes**") makes sense.